content/en/blog/2022/k8s-otel-expose.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Exposing an OpenTelemetry Collector for cross cluster communication
+linkTitle: Exposing an OpenTelemetry Collector
+date: 2022-08-25
+author: '[Benedikt Bongartz](https://github.com/frzifus)'
+spelling:
+  cSpell:ignore k8sattributes k8sattributesprocessor K8sattributes k8sprocessor
+  cSpell:ignore K8sprocessor KUBE
+---
+
+Exposing an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) currently requires a number of configuration steps. The goal of this blog post is to demonstrate "how to establish a secure communication" between two collectors in different kubernetes clusters.
+
+
+Details of CRDs and dependency installations are not covered by this post.
+
+# Overview
+
+The following graph illustrates the target topology. Goal is to transfer traces generated by a [test application](https://github.com/frzifus/jaeger-otel-test/pkgs/container/jaeger-otel-test) via a dedicated collector to a publicly accessible cluster. The receiving collector uses the transmitted ['Basic' HTTP Authentication](https://datatracker.ietf.org/doc/html/rfc7617) credentials to check whether the sender is authorized to store data. Finally, transmitted traces are stored in a [Jaeger in-memory](https://www.jaegertracing.io/docs/1.37/deployment/#memory) database for easy verification.
+
+```mermaid
```

Comment:
I am not sure, but I don't think that the blog supports turning mermaid into a picture (cc: @chalin), can you please replace this with a PNG?

Reply:
sure :)

Reply:
> I am not sure, but I don't think that the blog supports turning mermaid into a picture (cc: @chalin), can you please replace this with a PNG?

Yes, Docsy supports Mermaid diagram generation, see https://www.docsy.dev/docs/adding-content/diagrams-and-formulae/#diagrams-with-mermaid.

But using a PNG is fine too.